feature-group-definitions/compression-streams.yml

```
@@ -1,4 +1,5 @@
 name: Compression streams
+description: The JavaScript Compression Streams API compresses and decompresses streams of data using the gzip or deflate formats.
```

A couple of ideas here:

- I think this is a web API only, so I don't think it helps to mention JS.
- We can name the interfaces, since there aren't too many? (Less certain about the utility here.)

```suggestion
description: The `CompressionStream` and `DecompressionStream` interfaces compress and decompress data using the gzip or deflate formats.
```

---

I've run out of time, so this is a partial review—I haven't looked at any of the ones below.